benchmarks/gdscript/array.gd

```diff
@@ -1,14 +1,33 @@
 extends Benchmark
 
-const ITERATIONS = 80_000
-
+const ITERATIONS = 50_000
 
 # Benchmark various array types by:
 # 1) Inserting ITERATIONS elements from the back
 # 2) Updating all ITERATIONS elements
 # 3) Popping all ITERATIONS elements from the front (non-constant complexity)
 
-func benchmark_typed_int_array():
+func benchmark_packed_int32_array() -> void:
+	var array := PackedInt32Array()
+	for i in ITERATIONS:
+		array.push_back(i)
+	for i in ITERATIONS:
+		array[i] = 0
+	for i in ITERATIONS:
+		array.remove_at(0)
```

Comment:
Incidentally, was there a reason for popping from the front?

As-is, the quadratic complexity here means that the benchmark spends 25,000x more time removing elements than pushing/setting them, essentially turning this into a suite of `remove_at()` benchmarks

Reply:
I think I wanted to test quadratic complexity by design, but now that you mention it, it may not be a good idea to do so.

Reply:
One option would be to make them all quadratic:

```gdscript
	for i in ITERATIONS:
		array.push_front(i)
	for i in ITERATIONS:
		array.fill(0)
	for i in ITERATIONS:
		array.remove_at(0)
```

Reply:
I've made them all linear.

PS: I noticed packed arrays lack `pop_back()`, `pop_front()` and `pop_at()`.